packages/plugin-core/package.json

```diff
@@ -41,21 +41,6 @@
         "configuration": "./language-configuration.json"
       }
     ],
-    "viewsWelcome": [
```

Comment:
it looks like the new views weren't added to the config/constants. i know this isn't part of this feature but could you add them to the `gen:config` script?